app/controllers/admin/routes.py

```diff
@@ -119,6 +124,8 @@ def editEvent(eventId):
         eventData = request.form.copy()
         saveSuccess, validationErrorMessage = attemptSaveEvent(eventData)
         if saveSuccess:
+            event = Event.get(name=eventData['name'],term=eventData['term'],timeStart=eventData['timeStart'],startDate=eventData['startDate'])
```

Comment:
Can't we get the event using just the name and term? Or is there a possibility of having multiple events with the same name?

Also, would it be possible to have attemptSaveEvent to return the event id instead of boolean? Of course, if that messes up the overall logic then please ignore.

Reply:
I think it is possible to have events with the same name on the same semester. However, I can change the logic and we see what Brian thinks.

Reply:
I had this same thought, Hila. Since we can return multiple events from `saveEventToDb`, this wouldn't quite be accurate even if we made `attemptSaveEvent` return them. I could see an argument to having the first parameter `attemptSaveEvent` returns be an event array though. Ultimately my suggestion is that we should move the logging next to the actual creation of the event. That way you don't have to re-query the database, and the logging happens anytime an event is created without having to remember to add logging every time you call the function.